src/item.cpp

```diff
@@ -4903,6 +4907,28 @@ bool item::reload( player &u, item_location loc, long qty )
     return true;
 }
 
+bool item::is_tinder() const {
+    const auto &mats = made_of();
+    float fuel = 0.0f;
+    for( const auto &m : mats ) {
+        const auto &bd = m->burn_data( 1 );
+        if( bd.immune ) {
+            break;
+        }
+        fuel += bd.fuel;
+    }
+
+    float vol = base_volume().value() / 1000.0f;
```

Comment:
Please use something like `to_milliliter( base_volume() )` so a reader has a better understanding what range to expect here. And maybe change the name of `vol` to `vol_in_liter`.